scss/lessonsChallenges.scss

```diff
@@ -0,0 +1,47 @@
+.profile-submissions{
+
+  .lesson_challenges{
+    display: flex;
+    margin-top: 15px
+  }
+
+  .lessons_challenges_card_body{
+    margin-left: 30px;
```

Comment:
remove. Use the default, otherwise this will look different from your other component (ProfileLessons)